src/index.js

```diff
@@ -53,21 +53,19 @@ if (packageVersion !== lastPackageVersion) {
 function Providers() {
   const { appearance } = useClientTheme()
   return (
-    <Main layout={false} scrollView={false} theme={appearance}>
-      <HelpScoutProvider>
-        <ConsoleVisibleProvider>
-          <GlobalErrorHandler>
-            <ClientBlockNumberProvider>
-              <WalletBlockNumberProvider>
-                <WalletProvider>
-                  <App />
-                </WalletProvider>
-              </WalletBlockNumberProvider>
-            </ClientBlockNumberProvider>
-          </GlobalErrorHandler>
-        </ConsoleVisibleProvider>
-      </HelpScoutProvider>
-    </Main>
+    <WalletProvider>
+      <Main layout={false} scrollView={false} theme={appearance}>
```

Comment:
Interesting; I would have expected `Main` then `WalletProvider`.

It doesn't make a difference, but I'm wondering if it's still better to keep `Main` at the top just so we keep the invariant that it's always the first component to render when using aragonUI?

Reply:
Yes let’s do this for now. In the future, we might want to pass some things from use-wallet (and aragonAPI) to aragonUI, but we’ll reorder things as needed.